src/nft/components/details/AssetPriceDetails.tsx

```diff
@@ -312,6 +314,10 @@ export const AssetPriceDetails = ({ asset, collection }: AssetPriceDetailsProps)
 
   const USDPrice = useUsdPrice(asset)
 
+  const assetsFilter = [{ address: asset.address, tokenId: asset.tokenId }]
+  const { walletAssets: ownerAssets } = useNftBalanceQuery(account ?? '', [], assetsFilter, 1)
+  const walletAsset: WalletAsset = useMemo(() => ownerAssets[0], [ownerAssets])
```

Comment:
do we need a check to see if ownerAssets.length > 0?

Reply:
maybe walletAsset can be undefined

Reply:
It can be undefined which is what is checked for in the isOwner bool